### ISSUE TITLE
Add commenting & Social in gov4git app

### ISSUE THREAD

**Issue by KasiaSun** [gov4git/roadmap#13]
As a user of Gov4Git, I want the capability to post comments within the Gov4Git application. 

This feature would enable me to provide a rationale for my votes and the credits I utilize to prioritize specific issues. The inclusion of a social component in the voting mechanism would allow fellow users to comprehend the reasons behind my significant or minimal credit expenditure on a vote.

**Comment by gov4git-dao** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Monday, 22-Jan-24 22:13:20 UTC` by Gov4Git `dev`

### Notice `2qhlja`
Started managing this issue as Gov4Git concern `13` with initial __priority score__ of `0.000000`.

This project is managed by [Gov4Git](https://gov4git.org), a decentralized governance system for collaborative git projects.
To participate in governance, __install the [Gov4Git desktop app](https://github.com/gov4git/desktop-application)__.

**Comment by dworthen** [comment]
@petar, correct me if I am wrong but I believe we will need to update the backend first to support this feature. Marking this for backend work for now.

**Comment by petar** [comment]
@dworthen Yes, and more than that. This feature needs to be researched, designed, then backend, then frontend. I would view this as a major new feature (not short term) which requires a detailed discussion.
